### ISSUE TITLE
cannot find members of StandardValueSet by "sfdx force:source:manifest:create"

### ISSUE THREAD

**Issue by takahitomiyamoto** [forcedotcom/cli#1877]
### Summary
In order to create a package.xml automatically, I ran the command `sfdx force:source:manifest:create --apiversion 56.0 --fromorg demo -o manifest -t package`. When I checked the package.xml, I cannot find the tag `<name>StandardValueSet</name>` .

### Steps To Reproduce:

```
scoop update nvm
nvm version   # 1.1.10
nvm install 18.13.0
nvm use 18.13.0
node -v   # v18.13.0
npm -v   # 8.19.3
npm install -g sfdx-cli@7.183.1
sfdx -v   # sfdx-cli/7.183.1 win32-x64 node-v18.13.0
sfdx force:project:create -x -n demo
cd demo
sfdx force:source:manifest:create --apiversion 56.0 --fromorg demo -o manifest -t package
```

### Expected result

I can find the tag `<name>StandardValueSet</name>`  in the `manifest/package.xml` .

### Actual result

I cannot find the tag `<name>StandardValueSet</name>`  in the `manifest/package.xml` .

### System Information

- Which shell/terminal are you using? (e.g. bash, zsh, powershell 5, powershell 7, cmd.exe, etc.)

`PowerShell 7.3.1`

- If you are using `sfdx`
	- Run `sfdx version --verbose --json`
```
{
  "cliVersion": "sfdx-cli/7.183.1",
  "architecture": "win32-x64",
  "nodeVersion": "node-v18.13.0",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 1.3.8 (core)",
    "@oclif/plugin-commands 2.2.2 (core)",
    "@oclif/plugin-help 5.1.20 (core)",
    "@oclif/plugin-not-found 2.3.13 (core)",
    "@oclif/plugin-plugins 2.1.8 (core)",
    "@oclif/plugin-search 0.0.6 (core)",
    "@oclif/plugin-update 3.0.9 (core)",
    "@oclif/plugin-version 1.1.4 (core)",
    "@oclif/plugin-warn-if-update-available 2.0.18 (core)",
    "@oclif/plugin-which 2.2.6 (core)",
    "alias 2.1.14 (core)",
    "apex 1.3.0 (core)",
    "auth 2.3.8 (core)",
    "community 2.0.18 (core)",
    "config 1.4.22 (core)",
    "custom-metadata 2.0.5 (core)",
    "data 2.1.19 (core)",
    "generator 2.0.13 (core)",
    "info 2.3.1 (core)",
    "limits 2.2.0 (core)",
    "org 2.2.20 (core)",
    "packaging 1.12.2 (core)",
    "schema 2.2.2 (core)",
    "signups 1.2.12 (core)",
    "source 2.3.12 (core)",
    "telemetry 2.0.5 (core)",
    "templates 55.1.0 (core)",
    "trust 2.2.4 (core)",
    "user 2.1.24 (core)",
    "@salesforce/sfdx-plugin-lwc-test 1.0.1 (core)",
    "heat-sfdx-cli 0.3.3 (user)",
    "salesforce-alm 54.8.5 (core)",
    "sfdx-plugin-prettier 1.0.5 (user)",
    "sfpowerkit 5.0.0 (user)"
  ],
  "osVersion": "Windows_NT 10.0.22000",
  "shell": "cmd.exe",
  "rootPath": "C:\\Users\\takahito.miyamoto\\scoop\\persist\\nvm\\nodejs\\v18.13.0\\node_modules\\sfdx-cli"
}
```

### Additional information
By the way, when I use Org Browser in VS Code for the same sandbox, I can find the members of StandardValueSet.

![image](https://user-images.githubusercontent.com/22555172/212229367-94faa11c-d4d2-45e8-acd7-6edfb37b2f6f.png)

**Comment by WillieRuemmele** [comment]
Hi @takahitomiyamoto - I used the dreamhouse project, and then created a manifest with that command. I see 
```
        <members>TaskSubject</members>
        <members>TaskType</members>
        <members>WorkOrderLineItemStatus</members>
        <members>WorkOrderPriority</members>
        <members>WorkOrderStatus</members>
        <name>StandardValueSet</name>
    </types>
```
in the manifest. I would check your `.forceignore` file to see if maybe they're ignored?

**Comment by takahitomiyamoto** [comment]
@WillieRuemmele Thanks, but my `.forceignore` file was not changed from default. Please see here:

```
# List files or directories below to ignore them when running force:source:push, force:source:pull, and force:source:status
# More information: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_exclude_source.htm
#

package.xml

# LWC configuration files
**/jsconfig.json
**/.eslintrc.json

# LWC Jest
**/__tests__/**
```

Are you using a developer edition org or a sandbox org ?

**Comment by SabariRavi** [comment]
Hi ,

- This issue affected me too , with the version sfdx-cli/7.183.1 , we cannot reterive the StandardValueSet 
-  .forceignore , previusoly ignored quickaction , permission set and some roles too getting retirived. Not all the files are retrieved but some files.
![image](https://user-images.githubusercontent.com/30651882/212727554-6fe0d143-0311-4201-9391-c5b8e9760f0a.png)

**Comment by WillieRuemmele** [comment]
Hi @takahitomiyamoto - I was using a scratch org for this, I'll spin up a sandbox and test... Did this start happening after a specific release?

**Comment by shetzel** [comment]
UPDATE: This is fixed in the current CLI release candidate, v7.186.0 which will be promoted in a week.

**Comment by shetzel** [comment]
The fix is now in the current production CLI, v7.186.2.
